recipes/joommf/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "joommf" %}
+{% set version = "0.8" %}
+{% set sha256 = "b2a4b1fce0377318b47bc1dee1d2a5f36b32d22136716cc1dbfe5d437062912b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py27]
+  skip: True  # [win32]
```

Comment:
`noarch: python` can't have selectors, so remove these and in requirements change `python` to `python >= 3.5`